tests/test_integration_crashpad.py

```diff
@@ -114,7 +114,7 @@ def test_crashpad_proxy_set_empty(cmake, httpserver):
             cmake, httpserver, "http-proxy"
         )
 
-        with httpserver.wait(timeout=15) as waiting:
+        with httpserver.wait(timeout=30) as waiting:
```

Comment:
I don't think whatever is bugging you here is an httpserver timeout issue. I think it makes sense to revert these.